virtio-queue/src/queue.rs

```diff
@@ -269,6 +269,259 @@ impl Queue {
     }
 }
 
+#[cfg(kani)]
+#[allow(dead_code)]
+mod verification {
+    use std::mem::ManuallyDrop;
+    use vm_memory::MmapRegion;
+
+    use std::num::Wrapping;
+    use vm_memory::FileOffset;
+
+    use vm_memory::guest_memory::GuestMemoryIterator;
```

Comment:
I think we can't use `GuestMemoryIterator`. See https://github.com/rust-vmm/vm-memory/pull/278

Reply:
fixed